### ISSUE TITLE
Anything new regarding this mod?

### ISSUE THREAD

**Issue by Sunconure11** [A13XIS/dendrology-legacy-mod#21]
You seem to have ported to 1.12, yet you have nearly vanished. Any news on a new build for this mod?

**Comment by A13XIS** [comment]
Yes. It does not work and I have no idea why. According to the original author the internal changes in Minecraft 1.12 regarding Block registration are so severe, that I'd have to rewrite the mod almost from scratch. At least the Block Classes and the Block Creation/Registration mechanism, so I just stopped.

**Comment by Sunconure11** [comment]
Registries changed in 1.12 heavily.

https://mcforge.readthedocs.io/en/latest/concepts/registries/
